src/Features/Core/Portable/IntroduceVariable/AbstractIntroduceVariableService.cs

```diff
@@ -160,8 +166,8 @@ private string AddActionsAndGetTitle(State state, ArrayBuilder<CodeAction> actio
             else if (state.InExpressionBodiedMemberContext)
             {
                 CreateConstantFieldActions(state, actions, cancellationToken);
-                actions.Add(CreateAction(state, allOccurrences: false, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
-                actions.Add(CreateAction(state, allOccurrences: true, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
+                actions.Add(CreateAction(state, allOccurrences: false, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false, includeLValues: true));
+                actions.Add(CreateAction(state, allOccurrences: true, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false, includeLValues: true));
```

Comment:
this seems wrong.  i would think for expr-bodied members you would still want the lvalue rvalue split.  i.e. if i have `=> list[i] = list[i] + list[i]`, and i introduce local for `list[i]` then you would not want to update the lvalue version, only the two rvalue versions.